### DISCUSSION TITLE
GraphQL: For fields with values and labels, retrieve both

### DISCUSSION THREAD

**Discussion by bdnorris** [craftcms/cms#11540] [Ideas] [answered]
With Craft's GraphQL API, if my entry has field types that have values/labels (checkboxes, dropdowns, etc.) I can add `(label: true)` in order to retrieve the label instead of the value. However, there doesn't appear to be a way to get both the label and the value through GraphQL. 

This has led me to have to use tags for fields when a checkbox would probably be a better user interface. Am I missing something? If not, might this be possible to add?

**Comment by brandonkelly** [comment, marked as answer]
That’s currently expected. The front-end view would need to render its own labels based on the selected option values.
